tests/test_pca.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn.datasets import make_classification
+from matplotlib.testing.decorators import image_comparison
+
+from sklearn_evaluation.plot.pca import pca
+
+X, y = make_classification(
+    n_samples=5000, n_features=8, n_informative=6, class_sep=3.0, random_state=0
+)
+
+
+@image_comparison(baseline_images=["pca"], extensions=["png"], remove_text=False)
+def test_pca():
+    pca(X, y, colors=["r", "b"])
+
+
+@image_comparison(
+    baseline_images=["pca_without_y"], extensions=["png"], remove_text=False
+)
+def test_pca_without_y():
```

Comment:
we can add another test:  `test_pca_without_colors`

Reply:
Added but since colors are randomly generated in every run the `image_comparison` hasn't been added